test/integration/fdp-class.browser.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { join } from 'path'
+import { beeDebugUrl, beeUrl, generateUser } from '../utils'
+
+jest.setTimeout(200000)
+describe('Fair Data Protocol class - in browser', () => {
+  const BEE_URL = beeUrl()
+  const BEE_DEBUG_URL = beeDebugUrl()
+
+  beforeAll(async () => {
+    await jestPuppeteer.resetPage()
+    const testPage = join(__dirname, '..', 'testpage', 'testpage.html')
+    await page.goto(`file://${testPage}`)
+  })
+
+  it('should create a new FDP instance in browser', async () => {
+    const urls = await page.evaluate(
+      (BEE_URL, BEE_DEBUG_URL) => {
+        const accountData = new window.FairDataProtocol.FairDataProtocol(BEE_URL, BEE_DEBUG_URL).accountData
```

Comment:
create an import from the FDP's index file in order to get the type, because the IDE does not know about it like this.